backend/WSS/models.py

```diff
@@ -37,7 +51,10 @@ def __str__(self):
         return 'WSS {}'.format(self.year)
     
     def is_capacity_full(self, grade: str):
-        return Participant.objects.filter(current_wss=self, user_profile__grade=grade).count() >= getattr(self, f"{grade}_participant_limit")
+        if grade not in GRADE_GROUPS:
+            raise GradeDoesNotSpecifiedException()
```

Comment:
do we respond with a helpful message in this case?

Reply:
yes:
You must specify your grade in your profile before registration.